website/cue/reference/remap/errors/111_fallible_predicate.cue

```diff
@@ -0,0 +1,30 @@
+package metadata
+
+remap: errors: "111": {
+	title:       "Unhandled predicate error error"
+	description: """
+		A predicate is fallible and its [runtime error](\(urls.vrl_runtime_errors)) isn't handled in the VRL
+		program.
+		"""
+	rationale:   remap._fail_safe_blurb
```

Comment:
This is a placeholder, I suppose? 😄

Reply:
It's copied from here: https://github.com/vectordotdev/vector/blob/master/website/cue/reference/remap/errors/100_unhandled_root_runtime_error.cue#L9

I presume it's some cue thing..

Reply:
Now that's funny. The "blurb" just caught me off there 😹